html/includes/print-map.inc.php

```diff
@@ -84,7 +84,7 @@
                              `D1`.`device_id` != `D2`.`device_id`
                              $where
                              $sql
-                      GROUP BY `P1`.`port_id`,`P2`.`port_id`
```

Comment:
GROUP BY must contain all the fields in the SELECT. Tested and # results are the same.